pymc3/distributions/multivariate.py

```diff
@@ -2036,30 +2074,41 @@ def logp(self, value):
         TensorVariable
         """
 
+        sparse = isinstance(W, aesara.sparse.SparseVariable)
+
+        if sparse:
+            D = [
+                aesara.sparse.basic.csm_data(W[i : i + 1, :]).eval().sum()
+                for i in range(int(W.shape[0].eval()))
+            ]
```

Comment:
@brandonwillard , I added this as a workaround for the `.sum()` method, which is not implemented for a sparse variable. But maybe there is a more optimal solution?

Reply:
@brandonwillard is this something that should be implemented at the aesara level?

Reply:
Absolutely

Reply:
Is there an issue already?